config/vendor_api/draft.yml

```diff
@@ -154,591 +55,32 @@ paths:
-          description: Application status before the deferral (pending_conditions or recruited)
-          enum:
-          - pending_conditions
-          - recruited
-          maxLength: 30
-          example: pending_conditions
-        deferred_to_recruitment_cycle_year:
-          type: integer
-          nullable: true
-          description: The recruitment cycle year a deferred application can be confirmed
-          example: 2020
-        offer_deferred_at:
-          type: string
-          format: date-time
-          nullable: true
-          description: When this application was deferred
-          example: 2019-06-10T23:59:59Z
-    ConfirmDeferredOffer:
-      type: object
-      additionalProperties: false
-      required:
-      - conditions_met
-      properties:
-        conditions_met:
-          type: boolean
-          description: Determines the status of the confirmed deferred application (recruited vs. pending_conditions)
-          example: true
-    Links:
-      type: object
-      description: Links to navigate through the paginated results
-      additionalProperties: false
-      required:
-      - first
-      - last
-      - prev
-      - self
-      - next
-      properties:
-        first:
-          type: string
-          description: The first page of the results
-          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z&page=1
-        last:
-          type: string
-          description: The last page of the results
-          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z&page=20
-        prev:
-          type: string
-          description: The previous page of the results
-          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z&page=1
-        self:
-          type: string
-          description: The current page of the results
-          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z&page=2
-        next:
-          type: string
-          description: The next page of the results
-          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z&page=3
-    ResponseMetaMultiple:
-      type: object
-      description: API version and count metadata in multiple application responses
-      additionalProperties: false
-      required:
-      - api_version
-      - timestamp
-      - total_count
-      properties:
-        api_version:
-          type: string
-          description: API version that produced this response
-          example: 'v1.1'
-        timestamp:
-          type: string
-          format: date-time
-          description: When this response was generated
-          example: 2021-12-01T11:03:12Z
-        total_count:
+        code:
           type: integer
```

Comment:
might this be a string?

Reply:
Yeah string is definitely more flexible

Reply:
[Now a string](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7254/commits/9560905156ab9c380482eba1f0b16df04e8e2cb1#diff-409b89434fef4cef9730cceeef56883e0f6fd7ac8ec20fbaa2d10a251931d47dR79)